Updated Readme.md

tamerharsh opened the pull request:
Please review.

tamerharsh committed:
Updated Readme.md

Please review

aniket611 commented on README.md:
```diff
@@ -1,3 +1,107 @@
-Just Learning Git
+# CommandLineParser
+# Introduction
+[1].CommandLineParser library adds the user defined command line options which accepts the short command , long command , short description ,long description and a  default value.
```
Cosmetic: Please keep consistent spaces after punctuation marks like , and .

aniket611 commented on README.md:
```diff
@@ -1,3 +1,107 @@
+[5].Example 
+
+let say the, resgistred command is port . 
+ 
```
Avoid using sentences like "let say the ..". In technical documentation we generally we address third person.
Here we can write,
Port registration command
obj.............

aniket611 commented on README.md:
```diff
@@ -1,3 +1,107 @@
+ Result 
+    
+    The value for --port : 324 425
+
```
obj.Addoptions (Should we Camel case for function names? AddOptions)

aniket611 commented on README.md:
```diff
@@ -1,3 +1,107 @@
+In  CommandPrompt
+
+    SampleApp.exe --port=324 425
+ 
```
obj.Addoptions says port number must be 4 digits. But in example it seems to be different

aniket611 commented on README.md:
```diff
@@ -1,3 +1,107 @@
+     cmake -Ax64 -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="F:/Projects/CommandLine/install" -L ..
+	 cmake --build . --target install  
+This library has cmake build system which supports the find_package() functionlaity .
+
```
functionality

tamerharsh replied:
Please explain.

tamerharsh committed:
made changes-1

1. Made changes as per as comments.
2. Made the directory using cmake variables  instead of  fixed names example  set   cmd_lib_BUILD_DIR=C:\ instead of  giving instruction directly cd  C:\.

tamerharsh committed:
Update 3

tamerharsh committed:
Update 4

tamerharsh committed:
Update 5

tamerharsh committed:
Updated final
